js/exercise-00.js

```diff
@@ -0,0 +1,22 @@
+/* 
+   Create a constant variable of each of the following primitive types:
+   - String
+   - Number
+   - Boolean
+   - Null
+   - Undefined
+
+   Log out each variable to the terminal at the end.
+*/
+
+const String = 'string';
```

Comment:
This script will niot work. Have you tested it before pushing?

FIX: make this script properly works